### PR TITLE
[Port] Ports NSV13's rbmk fission reactor

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

Ports the rbmk fission reactor, fuel rods for it from nsv13 and also flex seal to fix it.
BeeStation/NSV13#828

## Why It's Good For The Game

*It's not 3 Roentgen, it's 15,000*

## Changelog
:cl:
add: Nanotrasen has assigned this station an experimental fission reactor using exotic gases to moderate
add: Added flex seals
/:cl:

### CODE REVIEW

**Comment by park66665** [comment]
I have no idea why this is happening when a tg coder is seemingly working on this too, but I guess that tg maintainers won't let the tg coder port rbmk to tg since tg is having a *lot* of PRs lately, and, honestly, tg has oranges. So, uh, whatever.

**Comment by TheFakeElon** [comment]
3.6 roentgens

**Comment by TheFakeElon** [comment]
Alright I read through everything, should work! we don’t have a use for plutonium as of yet but that could come later.

_chef recipe_

**Comment by jupyterkat** [comment]
image

**Comment by TheFakeElon** [comment]
Fuck yeah

**Comment by TheFakeElon** [comment]
Pipes look a bit messy but I’m not sure what a clean reactor setup looks like..

**Comment by jupyterkat** [comment]
the special pool tile is just fluff and I can’t find the interaction between it and the tile itself

**Comment by Moccha-Bee** [comment]
damm based

**Review by TheFakeElon — APPROVED** [review]
properly credited now, looks ready

File: code/__DEFINES/sound.dm [hunk 1]
```diff
@@ -8,11 +8,12 @@
 #define CHANNEL_AMBIENT_EFFECTS 1018
 #define CHANNEL_AMBIENT_MUSIC 1017
 #define CHANNEL_AMBIENT_BUZZ 1016
+#define CHANNEL_REACTOR_ALERT 1015 //austation -- reactor sounds
 
 //THIS SHOULD ALWAYS BE THE LOWEST ONE!
 //KEEP IT UPDATED
 
-#define CHANNEL_HIGHEST_AVAILABLE 1015
+#define CHANNEL_HIGHEST_AVAILABLE 1014 //austation -- reactor sounds
```
Comment (MCterra10) [inline code/__DEFINES/sound.dm:16]: what?
is this part of modularized code?
I don't see how this would not break things on its own.

Comment (jupyterkat) [reply code/__DEFINES/sound.dm:16]: You always have to change it, check the comments above

**Pull request merged**